src/samples/RigidBodyAcceleration.hpp

```diff
@@ -9,12 +9,18 @@ namespace base { namespace samples {
     {
         base::Time time;
 
-        /** Acceleration in m/s, world fixed frame of reference (East-North-Up) */
+        /** Linear acceleration in m/s2, world fixed frame of reference (East-North-Up) */
         base::Vector3d acceleration;
-	/** Covariance matrix of the acceleration
+	/** Covariance matrix of the linear acceleration
 	 */
         base::Matrix3d cov_acceleration;
 
+        /** Angular acceleration in rad/s2, world fixed frame of reference (East-North-Up) */
+        base::Vector3d angular_acceleration;
+	/** Covariance matrix of the angular acceleration
+	 */
+        base::Matrix3d angular_cov_acceleration;
```

Comment:
What about `cov_angular_acceleration` to match with [RBS](https://github.com/rock-core/base-types/blob/master/src/samples/RigidBodyState.hpp#L70)?